theme/lumo/vaadin-select-styles.html

```diff
@@ -114,6 +114,18 @@
         justify-content: flex-end;
       }
 
+      :host([right-aligned]) {
+        align-items: flex-end;
+      }
+
+      :host([dir="rtl"]) {
+        align-items: flex-end;
+      }
+
+      :host([dir="rtl"][right-aligned]) {
+        align-items: flex-start;
+      }
+
```

Comment:
Should these rather go to the [select overlay core styles](https://github.com/vaadin/vaadin-select/blob/16310fa423a93809e952c1a4c8536c8dfb9f5d93/src/vaadin-select-overlay.html#L12-L17) instead of theme specific styles?

Reply:
Right, done. Also moved the `bottom-aligned` class to the core styles.